src/loading/cob/data/value/cob_builtin.rs

```diff
@@ -231,12 +265,23 @@ impl CobBuiltin
             match number.as_f32_lossy() {
                 Some(num) => Ok(num),
                 None => {
-                    tracing::warn!("failed parsing builtin Val at {}; number failed to convert to f32",
-                        get_location(content).as_str());
+                    tracing::warn!(
+                        "failed parsing builtin Val at {}; number failed to convert to f32",
+                        get_location(content).as_str()
+                    );
```

Comment:
```suggestion
                    tracing::warn!("failed parsing builtin Val at {}; number failed to convert to f32",
                        get_location(content).as_str());
```
Spurious formatting change